### PR TITLE
RxSwift 3.0 Support

### PR DESCRIPTION
**Pull request by kierangraham**
* [x] Updates to support `RxSwift ~> 3.0`
* [x] Adds `CoreMotion` as a dependent framework in `.podspec`
* [x] Fixes some _'target overrides ... build setting'_ CocoaPods warnings
* [x] Bumps the version number to `0.2.0` in the `.podspec`

### CODE REVIEW

File: Example/RxCoreMotion_Example/DeviceMotionViewController.swift [hunk 1]
```diff
@@ -15,24 +15,37 @@ class DeviceMotionViewController: UIViewController {
...
     let disposeBag = DisposeBag()
     let coreMotionManager = CMMotionManager.rx_manager()
+    let pedometerManager = CMPedometer()
+    
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
+        
+//        pedometerManager.rxPedometer(from: Date())
```
Comment (esttorhe) [inline Example/RxCoreMotion_Example/DeviceMotionViewController.swift]: Is this commented out code needed for reference or something?

File: Pod/Classes/RxCoreMotion.swift [hunk 2]
```diff
@@ -139,40 +141,114 @@ extension CMMotionManager {
...
 // If the current device supports one of the capabilities, observable sequence will not be nil
 public struct MotionManager {
     public let acceleration: Observable<CMAcceleration>?
     public let rotationRate: Observable<CMRotationRate>?
     public let magneticField: Observable<CMMagneticField>?
     public let deviceMotion: Observable<CMDeviceMotion>?
-
-    private init(motionManager: CMMotionManager) {
-        if motionManager.accelerometerAvailable {
+//    public let pedometer:Observable<CMPedometer>?
```
Comment (esttorhe) [inline Pod/Classes/RxCoreMotion.swift]: Same as before

File: Pod/Classes/RxCoreMotion.swift [hunk 3]
```diff
@@ -139,40 +141,114 @@ extension CMMotionManager {
...
-        if motionManager.deviceMotionAvailable {
+        if motionManager.isDeviceMotionAvailable {
             self.deviceMotion = motionManager.rx_deviceMotion
         }
         else {
             self.deviceMotion = nil
         }
+        
+        
+//        if CMPedometer.isStepCountingAvailable() {
```
Comment (esttorhe) [inline Pod/Classes/RxCoreMotion.swift]: And here 😄

**Comment by kierangraham** [comment]
@esttorhe made those changes 👍

**Comment by esttorhe** [comment]
🎉 can't release the `Pod` though; I'm not part of the maintainers

maybe one of the @RxSwiftCommunity/contributors could help with that 😇

**Comment by carlosypunto** [comment]
I'm sorry for all user. I forgot all it. Thanks @esttorhe by the warning. There are new release and the pod has been pushed.

**Comment by ashfurrow** [comment]
Cool thanks @carlosypunto! Do you want to add @esttorhe and/or myself as owners to the pod? It'd be `pod trunk add-owner RxCoreMotion [email address]`.

**Comment by carlosypunto** [comment]
Of course @ashfurrow! Done

**Comment by esttorhe** [comment]
Good to have more people to bug when releasing a new pod. Thanks

**Comment by carlosypunto** [comment]
Thanks to you @esttorhe for warning for it. I love that we are a good team

**Pull request merged**